gulpfile.js

```diff
@@ -50,6 +56,12 @@ gulp.task('clean:after', (done) => {
     done();
 });
 
+gulp.task('copy:formatter', (done) => {
+    shelljs.cp('-r', `${dirs.hint}/formatter-html`, `${dirs.hint}/formatter-html-copy`);
```

Comment:
I'm not sure this is going to work as we expect. The build process will work and revamp the files, but these files are under node_modules and they aren't commited. The files that get pushed into the server are the ones under `dist` and  `src` (with some exceptions). So if I've understood how this works correctly, those changes will be lost. Maybe instead of copying it under `node_modules` it should be somewhere in a folder that gets commited to the production Git repo?